conftest.py

```diff
@@ -16,10 +16,14 @@
 # under the License.
 import hashlib
 import pytest
+import sys
 import os
-from collections import OrderedDict
+
+from pathlib import Path
 
 pytest_plugins = ["tvm.testing.plugin"]
+IS_IN_CI = os.getenv("CI", "") == "true"
```

Comment:
should this be "CI" in os.environ?

Reply:
resolved offline, Jenkins sets it to `true`